examples/css/examples.css

```
@@ -32,7 +32,7 @@ h1 .logo {
     background-color: #14171d;
 }
 
-.hljs {
+.hljs, .CodeMirror {
```

This needs height:auto.
```suggestion
.hljs, .CodeMirror {
    height: auto !important;
```